.github/workflows/benchmarks.yml

```diff
@@ -115,21 +125,15 @@ jobs:
           target-branch: main
           target-directory: .asv
 
-      - name: Compare HEAD with master if PR
+      - name: Run benchmarks
         if: github.event_name == 'pull_request_target'
-        continue-on-error: true # TODO: step failed sporadically while testing
-        run: |
-          asv continuous ${{ github.event.pull_request.base.sha }} ${{ github.event.pull_request.head.sha }}  | tee asv-cont-output.log  
```

Comment:
What is the difference between asv continuous and asv compare?

Reply:
the doc says 
> Run a side-by-side comparison of two commits for continuous integration.

Not sure what continuous integration here means. That's why I have used asv compare.